sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -74,29 +74,6 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils {
     checkAnswer(query, Row(1, 1) :: Row(1, 2) :: Row(1, 3) :: Nil)
   }
 
-  test("SPARK-7550: Support setting the right schema & serde when writing to Hive Metastore") {
```

Comment:
Removed this test case since all its contents are covered in refactored `DataSourceWithHiveMetastoreCatalogSuite`.